components/pipefy/pipefy.app.js

```diff
@@ -0,0 +1,63 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "pipefy",
+  methods: {
+    _getBaseUrl() {
+      return "https://app.pipefy.com";
+    },
+    _getHeaders() {
+      return {
+        Accept: "application/json",
+        Authorization: `Bearer ${this.$auth.token}`,
+      };
+    },
+    async _makeRequest(data, endpoint) {
+      const config = {
+        method: "POST",
+        url: `${this._getBaseUrl()}/${endpoint}`,
+        headers: this._getHeaders(),
+        data,
+      };
+      return (await axios(config)).data;
+    },
+    async _makeQueriesRequest(data = null) {
+      return await this._makeRequest(data, "queries");
+    },
+    async _makeGraphQlRequest(data = null) {
+      return await this._makeRequest(data, "graphql");
+    },
+    async createHook({ pipe_id, name, email, url, actions }) {
+      const data = {
+        query: `mutation { createWebhook(input: { pipe_id: ${pipe_id} name: "${name}" email: "${email}" url: "${url}" actions: ["${actions}"] headers: \"{\\\"token\\\": \\\"${this.$auth.token}\\\"}\" }) { webhook { id name } } }`,
```

Comment:
Can we re-indent this so that it can be easily modified in the future if necessary?

```js
const query = `
  mutation {
    createWebhook(input: {
      pipe_id: ${pipe_id}
      name: "${name}"
      email: "${email}"
      url: "${url}"
      actions: [${actions.map(i => `"${i}"`).join(", ")}]
      headers: "{\\\"token\\\": \\\"${this.$auth.token}\\\"}\"
    }
    {
      webhook {
        id
        name
      }
    }
  }
`;
```

Reply:
Same for all the GraphQL queries below

Reply:
Also, looks like the `email` field is not necessary, and we should remove it so that:
- We handle as little private data as necessary
- The user doesn't get bombarded with e-mails that they didn't request